### PR TITLE
Fix type in unused constant name

### PR DESCRIPTION
**Pull request by strk**

### CODE REVIEW

**Comment by bkcsoft** [comment]
LGTM

File: routers/repo/webhook.go [hunk 1]
```diff
@@ -23,9 +23,9 @@ import (
 )
 
 const (
-	HOOKS        base.TplName = "repo/settings/hooks"
-	HookNew     base.TplName = "repo/settings/hook_new"
-	ORG_HookNew base.TplName = "org/settings/hook_new"
+	Hooks      base.TplName = "repo/settings/hooks"
```
Comment (bkcsoft) [inline routers/repo/webhook.go]: Convention seems to prefix templates with `tpl` :)

**Review by bkcsoft — CHANGES_REQUESTED** [review]

**Comment by strk** [comment]
On Mon, Nov 07, 2016 at 10:58:37AM -0800, bkcsoft wrote:

> Convention seems to prefix templates with `tpl` :)

Good point ! (and good convention)
[typos 645b5f3]

**Review by bkcsoft — APPROVED** [review]

**Comment by bkcsoft** [comment]
LGTM

**Comment by andreynering** [comment]
LGTM

**Pull request merged**